appveyor.yml

```diff
@@ -12,6 +11,10 @@ branches:
     - master
 skip_tags: true
 
+environment:
+  GH_TOKEN:
+    secure: Ki5AJWygDYhzMJxl0b0rDx3bhAYmar2aPdwVHiai9IigqsvZpWHLeI3qpTiiaOWL
```

Comment:
Where does the `GH_TOKEN` come from?

Reply:
I am thinking about whether there will be any security issues.

Reply:
> Where does the `GH_TOKEN` come from?

It's the encrypted GitHub token (`AppVeyor Settings → Encrypt YAML`).

> I am thinking about whether there will be any security issues.

> Secure variables
> 
> When you work with OSS projects and you’d like to hide some sensitive data from everyone’s eyes you can use secure variables in appveyor.yml.
> 
> AppVeyor generates a unique encryption key for every account. To encrypt variable values go to Settings → Encrypt YAML page.
> 
> To use encrypted variable in appveyor.yml:
> ```
> environment:
>   my_variable:
>     secure: <encrypt_value>
> ```
> 
> “Secure” variables means you can safely put them into appveyor.yml that is visible to others. Other than that they are just regular environment variables in a build session that could be easily displayed in a build log by simple Get-ChildItem env:.
> 
> However, secure variables are not decoded during Pull Request builds which prevents someone from submitting PR with malicious build script displaying those variables. In more controlled environment through with a trusted team and private GitHub repositories there is an option on General tab of project settings to allow secure variables for PRs.

https://www.appveyor.com/docs/build-configuration/#secure-variables

Reply:
great! that's OK!